templates/prescription/general_edit.html

```diff
@@ -258,8 +273,33 @@
 </form>
 {literal}
 <!-- for the fancy jQuery stuff -->
+<script type="text/javascript" src="./library/js/jquery-1.9.1.min.js"></script>
+<script type="text/javascript" src="./library/js/jquery-1.6.4.min.js"></script>
+<script type="text/javascript" src="./library/js/jquery-ui.min.js"></script>
+
+<script type="text/javascript" charset="utf-8" src="./library/js/prism.js" ></script>
```

Comment:
Let me know if we need the prism.js in the codebase. If so, then I'll add it to public/assets/